pkg/controller/datadogagentdeployment/agent_test.go

```diff
@@ -24,16 +24,37 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 )
 
-func Test_newExtendedDaemonSetFromInstance(t *testing.T) {
-	logf.SetLogger(logf.ZapLogger(true))
-	authTokenValue := &corev1.EnvVarSource{
-		SecretKeyRef: &corev1.SecretKeySelector{},
+var (
+	dadName = "foo"
+
+	authTokenValue = &corev1.EnvVarSource{
```

Comment:
Much cleaner overall, just one maybe dumb question. Why some default are vars and others functions? For consistency sake, I'd put everything function. Minor detail though.

Reply:
Good point, thank you, let me do that.